tests/test.txt

```diff
@@ -80,16 +80,16 @@ tests/testSpline.py
 #tests/testStatistics.py
```

Comment:
Should tests/testSourceMatch.py be uncommented? I know that there is one test that doesn't run properly yet, so could that test be wrapped with a ```@unittest.skip("Skip test until pickling is fixed")``` decorator so that we ensure that nothing else breaks the other tests in it that pass?